frontend/src/components/form/base/ETextField.vue

```diff
@@ -6,11 +6,13 @@
     :vid="veeId"
     :rules="veeRules"
     class="e-form-container"
+    :required="required"
```

Comment:
I think there's no property named `required` on `ValidationProvider`:
https://vee-validate.logaretm.com/v3/api/validation-provider.html#props

What is this expected to do?

Reply:
See above, necessary for the asterisk. In the meantime, I added it to all e-inputs.